apps/editors/views.py

```diff
@@ -342,10 +351,18 @@ def save_motd(request):
     if form.is_valid():
         set_config('editors_review_motd', form.cleaned_data['motd'])
         return redirect(reverse('editors.motd'))
-    data = context(form=form)
+    data = context(request, form=form)
     return render(request, 'editors/motd.html', data)
 
 
+def is_admin_reviewer(request):
+    return acl.action_allowed(request, 'ReviewerAdminTools', 'View')
+
+
+def exclude_admin_only_addons(queryset):
+    return queryset.filter(admin_review=False, has_info_request=False)
```

Comment:
It's for the need info flag. That was what was suggested to filter on in https://bugzilla.mozilla.org/show_bug.cgi?id=1109299#c2.